phpunit.xml.dist

```diff
@@ -45,7 +45,13 @@
 		<file>tests/Backend/Workspaces/WorkspacesUnloadTest.php</file>
 		<file>tests/Backend/Workspaces/WorkspacesLoadTest.php</file>
 		<file>tests/Backend/Workspaces/WorkspacesRenameLoadTest.php</file>
+	</testsuite>
+
+	<testsuite name="paratest-backend-snowflake-abs-part-1">
 		<directory>tests/Backend/Snowflake</directory>
+		<exclude>tests/Backend/Snowflake/CloneIntoWorkspaceTest.php</exclude>
```

Comment:
Testy, které jsou tady excludnuté nejsou přidané v `	<testsuite name="backend-snowflake-abs-part-1">` to je intentional nebo chyba?

Reply:
Jj to mi vypadlo, to tam má samozřejmě být přidané tyhle testy. Opravím.

Reply:
fixed